### ISSUE TITLE
Python 3.10

### ISSUE THREAD

**Issue by holgerpieta** [KaSroka/Toshiba-AC-control#12]
The day before yesterday I found about that project and tried it and it worked. Nice.
Yesterday I tried it again and it does not work anymore. Only thing I changed (knowingly) is the Python version: Previously I had 3.9, now I have 3.10. So maybe something isn't working with the new version.

The debug output indicates that something is happening, but then it doesn't go on:
```
[2021-10-31 17:36:31,189] DEBUG    toshiba_ac.http_api: Sending POST to https://toshibamobileservice.azurewebsites.net/api/Consumer/Login with params=None, json={'Username': '*****', 'Password': '*****'}, headers={'Content-Type': 'application/json'}
[2021-10-31 17:36:31,385] DEBUG    toshiba_ac.http_api: Response code: 200, JSON: {'ResObj': {'access_token': '*****', 'token_type': 'bearer', 'expires_in': 1576799999, 'consumerId': '*****', 'countryId': 85, 'consumerMasterId': '*****'}, 'IsSuccess': True, 'Message': 'Success', 'StatusCode': 'Success'}
[2021-10-31 17:36:31,386] DEBUG    toshiba_ac.http_api: Sending POST to https://toshibamobileservice.azurewebsites.net/api/Consumer/RegisterMobileDevice with params=None, json={'DeviceID': '*****', 'DeviceType': '1', 'Username': '*****'}, headers={'Content-Type': 'application/json', 'Authorization': 'bearer *****'}
[2021-10-31 17:36:31,490] DEBUG    toshiba_ac.http_api: Response code: 200, JSON: {'ResObj': {'DeviceId': '*****', 'HostName': 'toshibasmaciothubprod.azure-devices.net', 'PrimaryKey': '*****', 'SecondaryKey': '*****=', 'SasToken': 'SharedAccessSignature sr=toshibasmaciothubprod.azure-devices.net%2Fdevices%2F*****&sig=*****&se=*****', 'RegisterDate': '2021-10-27T19:07:23.627'}, 'IsSuccess': True, 'Message': 'Success', 'StatusCode': 'Success'}
```

The program window looks just like this:
![grafik](https://user-images.githubusercontent.com/38852807/139593742-3ac462ef-3f32-4a6e-ad5e-5f0cd2ac50ec.png)

There were some changes in the asyncio between Python 3.9 and 3.10, so maybe that broke something.

I'll try to go back to 3.9 to test the hypothesis, but maybe you want to test 3.10, too.

**Comment by holgerpieta** [comment]
Ah, well, I couldn't let that rest so I dug a bit deeper:
Janus < 0.6.2 doesn't work with Python 3.10, so we need 0.6.2.
azure-iot-device (even in the current version 2.8.0) requires Janus 0.4.0, so that's what's installed automatically.
Manually installing Janus 0.6.2 works, but of course gives a requirement conflict warning.

Nevertheless, everything seems to work, at least in a quick test, with a manually installed Janus 0.6.2.
So maybe you could solve the problem by updating setup.cfg with janus==0.6.2.
But I do not know if that then still works with Python < 3.10.

**Comment by KaSroka** [comment]
Thanks for reporting this. I'll check if we can update to azure-iot-device that uses new Janus (if such version exists)

**Comment by holgerpieta** [comment]
azure-iot-device already fixed that: https://github.com/Azure/azure-iot-sdk-python/issues/891
Just pull in a version >= 2.9.0 and we're fine.
By the way, I'm already using 2.9.0 and it's been working flawlessly for weeks.

**Comment by KaSroka** [comment]
Fixed in 0.1.13 by #16
